spec/acceptance/rds_spec.rb

```diff
@@ -92,6 +92,12 @@ def get_rds_instance(name)
       end
     end
 
+    it 'with the default subnet association (VPC-only accounts)' do
+      unless @aws.vpc_only?
+        expect(@rds_instance.db_subnet_group.db_subnet_group_name).to eq('default')
```

Comment:
I've reformulated this as a single test rather than two tests. The test above contained the reverse expectations.